routes/web.php

```diff
@@ -19,5 +19,5 @@
 });
 Route::get('language/{language}', [LanguageController::class, 'index'])->name('language');
 Route::resource('/task', tasksControllers::class)->only([
-    'index', 'store'
+    'index', 'store', 'destroy'
 ]);;
```

Comment:
```suggestion
    'index', 'store', 'destroy',
]);
```
với array viết trên nhiều dòng thì phần tử cuối phải có dấu phẩy nhé.